docs/src/modules/java/pages/views.adoc

```diff
@@ -452,15 +456,116 @@ Use the following filter predicates to further refine results:
 * `<` less than
 * `\<=` less than or equals
 
-Combine filter conditions with the `AND` and `OR` operators.
+==== Logical operators
+
+Combine filter conditions with the `AND` and `OR` operators, and negate using the `NOT` operator. Group conditions using parentheses. Note that `AND` has precedence over `OR`.
```

Comment:
don't we reject OR until that index issue has been fixed?

Reply:
removed OR